Helper/Parser.php

```diff
@@ -104,6 +108,16 @@ public function parseText($text, $checkTable = true) {
 
     $text = str_replace("#", "\\#", $text);
     $text = str_replace("_", "\\_", $text);
+    $text = str_replace("%", "\\%", $text);
+    $text = str_replace("$", "\\$", $text);
```

Comment:
Same holds for the $. I do not mind the #, but those should also be included in the option.